src/components/MdText.vue

```diff
@@ -0,0 +1,43 @@
+<script>
+import MarkdownIt from 'markdown-it'
+
+export default {
+    name: 'MdText',
+    props: {
+        source: {
+            type: String,
+            required: true
+        },
+        tag: {
+            type: String,
+            default: 'div'
+        }
+    },
+    data() {
+        return {
+            md: null
+        }
+    },
+    computed: {
+        content() {
+            const src = this.source
+            return this.md?.render(src)
+        }
+    },
+    created() {
+        this.md = new MarkdownIt()
+        this.md.options.breaks = true
+    },
+    destroyed() {
+        this.md = null
+    },
+    render(h) {
+        const paragraphs = this.content.replaceAll('<br>', '</p><p>')
```

Comment:
Can use two newlines for paragraph changes in Markdown instead.

Reply:
I guess if I change the locale messages file into js, I could use two new lines, as you say.

Reply:
Done! Now the messages are in a `js` format, so we can use formatted strings there.
Didn't realize that two new lines add a paragraph in Markdown :)